nuxt.config.js

```diff
@@ -36,6 +36,7 @@ export default {
     buildModules: [
         // Doc: https://github.com/nuxt-community/nuxt-tailwindcss
         '@nuxtjs/tailwindcss',
+        '@nuxtjs/google-analytics',
```

Comment:
Maybe throw a comment above this with a link to the docs, just for handy reference?

Reply:
Done!